tasks/install.yml

```diff
@@ -8,8 +8,13 @@
     - apt-transport-https
     - ca-certificates
     - curl
-    - "linux-image-extra-{{ ansible_kernel }}"
-    - linux-image-extra-virtual
+    - thin-provisioning-tools
```

Comment:
Do we want to install this if we are not using thin pool?

Reply:
It makes little difference, ultimately.. 
The extra code to make installation conditional is probably worse than just blanket installing it.

Reply:
Why? It would just be this:

```YAML
- name: Install thin-provisioning-tools is required
  become: yes
  apt:
    name: thin-provisioning-tools
  when: docker.thin_pool_device is not none```